apps/pie-docs/src/patterns/haptic-feedback/guidance/guidance.md

```diff
@@ -0,0 +1,139 @@
+---
+eleventyNavigation:
+    key: Guidance
+    parent: 'Haptic Feedback'
+    order: 2
+shouldShowContents: true
+---
+
+## About our haptic feedback types
+
+Our entire range of haptic feedback options relies on the system-default haptics provided by Apple in accordance with their [Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines).
+
+{% notification {
+  type: "warning",
+  message: "Please note that our default haptic feedback types **cannot be overridden or altered**."
+} %}
+
+---
+
+## Soft haptic feedback
+
+Soft haptic feedback can serve various purposes in order to enhance the overall user experience in a subtle and nuanced manner.
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/haptic-feedback/soft-haptic-feedback.svg",
+  alt: "Vertical short orange bar intersected by a horizontal line, representing soft haptic feedback signal.",
+  width: "344px"
+} %}
+
+#### Where can I use soft haptic feedback?
+
+{% list {
+    type: listTypes.icon,
+    iconName: "check-circle",
+    items: [
+        "**Button presses:** Provide a gentle confirmation for various button presses to acknowledge user input without being intrusive.",
+        "**Toggle and state changes:** Convey changes in toggle switches or activation of different states in the UI with a moderately pronounced haptic sensation.",
+        "**Dynamic adjustments:** Provide users with tactile feedback when making dynamic adjustments, such as using sliders, to indicate significant changes.",
+        "**Non-critical notifications:** Use soft haptic feedback for non-critical notifications to gently alert users without causing disruption.",
+        "**Subtle transitions:** Enhance subtle transitions or animations between UI elements for a smooth and cohesive experience.",
+        "**Incremental adjustments:** Offer soft haptic feedback for incremental adjustments, like scrolling or volume control, to provide a sense of granularity."
+    ]
+} %}
+
+---
+
+## Medium haptic feedback
+
+A medium-intensity haptic feedback which can convey a sense of importance and grab the user's attention. A good example of this might be the haptic feedback that occurs when a user navigates to a new screen, commonly known as a "screen transition haptic", which is designed to provide a tactile confirmation of the navigation action.
```

Comment:
```suggestion
Medium-intensity haptic feedback can convey a sense of importance and grab the user's attention. A good example of this might be the haptic feedback that occurs when a user navigates to a new screen, commonly known as a "screen transition haptic", which is designed to provide a tactile confirmation of the navigation action.
```